src/app/modules/group/components/group-log-view/group-log-view.component.html

```diff
@@ -35,15 +35,22 @@
           pTemplate="body"
```

Comment:
UI issue reported by @smadbe 
This case seems not to be handled correctly in terms of styling: (1st row of 1st test case)
![screenshot](https://user-images.githubusercontent.com/1053150/167797060-9cb65bb4-bdc4-4b9a-99cf-e912f8d8ba61.png)